src/index.ts

```diff
@@ -9,6 +9,8 @@ import {resolve} from './resolver'
 import {createServer} from './server'
 import {createWebhookProxy} from './webhook-proxy'
 
+// tslint:disable:no-var-requires
+// These needs types
```

Comment:
I started doing types for this in https://github.com/probot/probot/pull/565/files#diff-4f9fc8f2c9fb35fd5dafd4f84d420f82.

Reply:
I think probably makes the most sense if we put those types in https://github.com/octokit/webhooks.js and `import` them into Probot. Thoughts?